Source/NavierStokes.cpp

```diff
@@ -478,7 +478,9 @@ NavierStokes::initData ()
     }
 
 #ifdef AMREX_PARTICLES
-    initParticleData ();
+    if (do_nspc) {
```

Comment:
Do we really need this if you've added `if (!do_nspc) { return; }` in NSB::initParticleData? I suppose it doesn't really matter though.

Reply:
Right. It does not matter.